src/components/dashboard/widget/dashboard-widget.component.ts

```diff
@@ -80,6 +80,9 @@ export class DashboardWidgetComponent implements OnInit, AfterViewInit, OnDestro
     /** Defines a function that returns an aria label for the widget */
     @Input() widgetAriaLabel: (widgets: DashboardWidgetComponent) => string | string = this.getDefaultAriaLabel;
 
+    /** Defines whether or not this widget can be moved. */
+    @Input() canMove: boolean = true;
```

Comment:
This should be coerced to boolean, see https://github.houston.softwaregrp.net/caf/ux-aspects-team/wiki/Tips-and-tricks#coerce-inputs.